mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactTableView.js

```diff
@@ -0,0 +1,124 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { getSrc } from './ShowArtifactPage';
+import { getArtifactContent } from '../../../common/utils/ArtifactUtils';
+import { Table } from '@databricks/design-system';
+import { FormattedMessage } from 'react-intl';
+import Papa from 'papaparse';
+
+const ShowArtifactTableView = ({ runUuid, path, getArtifact }) => {
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState();
+  const [data, setData] = useState();
```

Comment:
I am curious if we could use the [RequestStateWrapper](https://github.com/mlflow/mlflow/blob/4c0f21025322f4681e5c4cf01d2ad57e0eb93115/mlflow/server/js/src/common/components/RequestStateWrapper.js) here rather than dealing with loading and error states. I think we can let this component handle that for us. What do you think?

Reply:
I will take a crack at this before merging!

Reply:
@sunishsheth2009 Is this just for network requests? There are two main errors that could occur with this component: 
1. A call to download the artifact from the artifact server which has its own function wrapped around the API call
2. An error parsing the CSV file in the browser which does not call to the server

Is this wrapper still suitable for these errors and loading?

Reply:
I think if we want to use [RequestStateWrapper](https://github.com/mlflow/mlflow/blob/4c0f21025322f4681e5c4cf01d2ad57e0eb93115/mlflow/server/js/src/common/components/RequestStateWrapper.js) we need to use redux pattern to use request_ids to work correctly. If we feel that we don't want to use redux here then I think we are fine leaving this approach as it is.
cc @harupy @hubertzub-db thoughts?

Reply:
I think it's fine to leave it as it is for now - looking at the forthcoming plans for improving the API connectivity code and restructuring the data store, there will be an opportunity to unify it anyway.

Reply:
Sounds good, all. That’s all the changes left on my end!